index.html

```diff
@@ -52,17 +52,20 @@ <h1>Leo de Andrade</h1>
         </div>
         <!-- end mobile version -->
 
-        <div class="intro">
-            <h1>Leo de Andrade</h1>
-            <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
-            <button title="Start Collaboration" class="btnstart" onclick=" window.open('#get-started', '_self'); return false;">Start Collaboration</button>
-        </div>
-        <div class="dots">
-            <img src="images/image_dots.svg" alt="">
-        </div>
-        <div class="image-geometri1">
-            <img src="/images/image-geometry_1.svg" alt="">
+        <div class="hello">
+            <div class="intro">
+                <h1>Leo de Andrade</h1>
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
+                <button title="Start Collaboration" class="btnstart" onclick=" window.open('#get-started', '_self'); return false;">Start Collaboration</button>
+            </div>
+            <div class="dots">
+                <img src="images/image_dots.svg" alt="">
+            </div>
+            <div class="image-geometri1">
+                <img src="/images/image-geometry_1.svg" alt="">
+            </div>
         </div>
+
     </section>
```

Comment:
- [ ] Kindly, refer to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) to implement the following as well as fix some issues:

#### Focus on the following:

* Work on the menu links to look the same as we have on the figma like (Changing the color, Remove the underline and remove the line under the menu link)
* Add the background image as requested.
* Try to reduce the size or the max-width as well as some width to look the same as we have on the figma.
*  
    ----------------------------------Original---------------------------------------

    
    ![header1](https://user-images.githubusercontent.com/55635977/126670472-5e063ef4-d6f2-4506-b1b4-4d0eaa8feccb.png)


    ----------------------------------Yours------------------------------------------

![header2](https://user-images.githubusercontent.com/55635977/126670407-8e8560c8-0544-4141-85e2-6646d336eca8.png)

Reply:
This is a Github bug. Everything was working fine early.

Reply:
Hi @leonardoeng13 sure?

Reply:
Fixed,